src/db/z_db.erl

```diff
@@ -233,7 +243,7 @@ assoc_props_row(Sql, Parameters, Context) ->
 
 
 %% @doc Return property lists of the results of a query on the database in the Context
-%% @spec assoc(SqlQuery, Context) -> Rows
+-spec assoc(string(), #context{}) -> list().
```

Comment:
The spec in line 246 uses a record, please define a type for the record and use that instead.